IOPool/Common/test/test_overlap_lumi_fast_copy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+# Pass in name and status
+function die { echo $1: status $2 ;  exit $2; }
+
+printenv
```

Comment:
Is this printout necessary for the test?

Reply:
removed